source/configure/configuration-settings.rst

```diff
@@ -26,14 +26,6 @@ Mattermost configuration settings are organized into the following categories wi
 
 In self-hosted Mattermost deployments, configuration settings are maintained in the ``config.json`` configuration file, located in the ``mattermost/config`` directory, or `stored in the database <https://docs.mattermost.com/configure/configuation-in-a-database.html>`__. System Admins managing self-hosted workspaces can also modify the ``config.json`` file directly using a text editor.
 
-Configuration in database
```

Comment:
Please revert this deletion, then update the first sentence to: "Self-hosted system configuration can be stored in the database."